packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -642,6 +642,160 @@ describe('RichTextEditor', () => {
                 'P'
             ]);
         });
+
+        describe('Absolute link interactions in the editor', () => {
```

Comment:
Should have a test for URL starting with just ``http`` (all these tests use ``https``).

Reply:
Done